string-manipulation-2/src/first-chars.js

```diff
@@ -0,0 +1,19 @@
+/* exported firstChars */
+function firstChars(length, string) {
+  var shortened = '';
+  if (length < string.length) {
+    for (var i = 0; i < length; i++) {
+      shortened += string[i];
+    }
+  } else {
+    for (var j = 0; j < string.length; j++) {
+      shortened += string[j];
+    }
```

Comment:
In this case, you want to `return` the entire string, so this can be simplified to `shortened = string`